src/UI/class-plugins-actions.php

```diff
@@ -39,13 +39,25 @@ public function run(): void {
 	 * @return array
 	 */
 	public function add_plugin_meta_links( array $actions ): array {
-		$settings_link = sprintf(
+		if ( is_multisite() && is_plugin_active_for_network( plugin_basename( PARSELY_FILE ) ) ) {
```

Comment:
Not a blocker, but would be nice (in another PR) to see the `PARSELY_FILE` global injected via a constructor arg, so we reduce the class's dependency on a global constant.